.env.example

```diff
@@ -10,7 +10,8 @@ OPENAI_API_KEY=''
 # DO NOT TRACK
 SCARF_NO_ANALYTICS=true
 DO_NOT_TRACK=true
+ANONYMIZED_TELEMETRY=false
 
 # Use locally bundled version of the LiteLLM cost map json
 # to avoid repetitive startup connections
-LITELLM_LOCAL_MODEL_COST_MAP="True"
```

Comment:
This should be set to "True" I believe.

Reply:
https://github.com/BerriAI/litellm/blob/285a3733a968bbf0c94e76d0ff5d61a80e7429af/litellm/__init__.py#L232